apps/site/util/blog.ts

```diff
@@ -12,3 +14,66 @@ export const mapBlogCategoryToPreviewType = (type: string): BlogPreviewType => {
       return 'announcements';
   }
 };
+
+export const getBlogPosts = (category: BlogCategory): BlogPostsRSC => {
+  const categoryPosts = blogData.posts.filter(post =>
+    post.categories.includes(category)
+  );
+
+  // Total amount of possible pages given the amount of blog posts
+  const total = categoryPosts.length / BLOG_POSTS_PER_PAGE;
+
+  return {
+    posts: categoryPosts,
+    pagination: {
+      prev: null,
+      next: null,
+      // In case the division results on a remainder we need
+      // to have an extra page containing the remainder entries
+      pages: Math.floor(total % 1 === 0 ? total : total + 1),
```

Comment:
Can this be simplified as:

```suggestion
      pages: Math.ceil(total),
```